playbooks/robusta_playbooks/job_actions.py

```diff
@@ -236,6 +236,23 @@ def job_info_enricher(event: JobEvent):
     event.add_enrichment([table_block])
 
 
+@action
+def delete_job(event: JobEvent):
+    """
+    Delete the job from the cluster
+    """
+    job = event.get_job()
+    if not job:
+        logging.info("Cannot run delete_job with no job. skipping")
+        return
+
+    # After deletion the metadata is empty. Saving the name and namespace
+    name = job.metadata.name
+    namespace = job.metadata.namespace
+    job.delete()
```

Comment:
FYI kubernetes could return an error here in job.delete, I think it might be printed in the logs so i don't think you need to add anything here
```
      if not 200 <= res.code <= 299:
          raise KubernetesException("Kubernetes returned error " + str(res.code))
```
maybe if it comes up we can investigate and add an enricher like "Job not deleted because of X"